src/ethereum/prague/utils/address.py

```diff
@@ -39,9 +41,35 @@ def to_address(data: Union[Uint, U256]) -> Address:
     return Address(data.to_be_bytes32()[-20:])
 
 
-def compute_contract_address(address: Address, nonce: Uint) -> Address:
+def to_address_without_mask(data: Union[Uint, U256]) -> Address:
+    """
+    Convert a Uint or U256 value to a valid address (20 bytes).
+    Raises a `ValueError` if the data is larger than `MAX_ADDRESS_U256
+
+    Parameters
+    ----------
+    data :
+        The string to be converted to bytes.
+
+    Raises
+    ------
+    ValueError
+        If `data` is larger than `MAX_ADDRESS_U256`.
+
+    Returns
+    -------
+    address : `Address`
+        The obtained address.
+    """
+    if data > MAX_ADDRESS_U256:
+        raise ValueError("Address is too large")
+    return Address(data.to_be_bytes32()[-20:])
```

Comment:
Would this be more clear if we used `to_be_bytes`, `left_pad_zero_bytes`, and `Address.LENGTH`?